docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -2,52 +2,52 @@
-
-Domain-1 to Domain-N are z/OS systems with the standard Zowe API ML running either in HA (sysplex) or non-HA (monoplex). These API MLs are referred to as Domain API MLs.
+- [Multitenancy Configuration](#multitenancy-configuration)
+  - [Overview of API MLs](#overview-of-api-mls)
+  - [Multitenancy component enablement settings](#multitenancy-component-enablement-settings)
+  - [Onboarding a Gateway service in one domain to the Discovery service of API ML in another domain](#onboarding-a-gateway-service-in-one-domain-to-the-discovery-service-of-api-ml-in-another-domain)
+    - [Dynamic configuration via zowe.yaml](#dynamic-configuration-via-zoweyaml)
+    - [Dynamic configuration via Environment variables](#dynamic-configuration-via-environment-variables)
+    - [Validating successful configuration](#validating-successful-configuration)
+  - [Establishing a trust relationship between the API MLs](#establishing-a-trust-relationship-between-the-api-mls)
+    - [Commands to establish trust between the API MLs](#commands-to-establish-trust-between-the-api-mls)
+  - [Using the `/registry` endpoint in the Gateway](#using-the-registry-endpoint-in-the-gateway)
+    - [Configuration for `/registry`](#configuration-for-registry)
+    - [Authentication for `/registry`](#authentication-for-registry)
+    - [Authorization with `/registry`](#authorization-with-registry)
+    - [Requests with `/registry`](#requests-with-registry)
+    - [Response with `/registry`](#response-with-registry)
+    - [Response with `/registry{apimlId}`](#response-with-registryapimlid)
+    - [Response with `GET /gateway/api/v1/registry/{apimlId}?apiId={apiId}&serviceId={serviceId}`](#response-with-get-gatewayapiv1registryapimlidapiidapiidserviceidserviceid)
+  - [Validating successful configuration with `/registry`](#validating-successful-configuration-with-registry)
+  - [Troubleshooting multitenancy configuration](#troubleshooting-multitenancy-configuration)
+    - [ZWESG100W](#zwesg100w)
+    - [No debug messages similar to apiml1 completed with onComplete are produced](#no-debug-messages-similar-to-apiml1-completed-with-oncomplete-are-produced)
+
+## Overview of API MLs 
+
+The following diagram illustrates communication between the API Mediation Layers and Zowe in multiple domains. Note that some API MLs may be running in a sysplex (HA), while others may be in a monoplex (non-HA).
+
+![Multi-domain architecture diagram](./diagrams/multi-domain_architecture_V3.svg)
+
+ As represented in the example diagram of Multitenacy environement where the APIMLs in Domain(2-N) are registered to APIML in Domain-1. The APIML in Domain-1 is running may be on z/OS, or off z/OS, for example in Kubernetes, this API ML serves as a single point of access to all API Mediation Layers registered in this and, by extension, to all services registered in those API MLs .
```

Comment:
Update to one of the following, please:
Domain-1 is running on z/OS, or off z/OS
Domain-1 may be running on z/OS, or off z/OS